### PR TITLE
New homepage design

### PR DESCRIPTION
**Pull request by megaslug**
homepage-redesign

### CODE REVIEW

**Comment by clippy** [comment]
**I have manually checked the visuals in:**

- [ ] Chrome
- [ ] IE11
- [ ] Firefox
- [ ] Safari
- [ ] iPad

**Comment by megaslug** [comment]
@jamesshedden could you review my branch, please?

File: app/css/components/hero-card.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  background-color: white;
+  flex-grow: 1;
+  border-radius: $border-radius-base;
+  padding: $space-xlarge $space-xmedium;
+  transition: box-shadow 0.15s,
+              transform 0.2s;
+  margin-bottom: $space-medium;
+  box-shadow: inset 0 0 0 1px rgba($gray, 0.68);
+
+  .btn {
```
Comment (jamesshedden) [inline app/css/components/hero-card.scss]: @megaslug Could we make this `&__btn` and then use the `hero-card__button` class in the template?

File: app/css/components/page-hero.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .page-hero {
   background: $primary-color;
-  padding-top: 76px;
+  padding-top: 0;
```
Comment (jamesshedden) [inline app/css/components/page-hero.scss]: @megaslug Same Q re: whether we can safely change this across the whole repo? May be affecting lots of other pages I think 🤔

File: app/css/components/page-hero.scss [hunk 3]
```diff
@@ -23,12 +22,12 @@
 }
 
 .page-hero__text__desc {
-  max-width: 650px;
+  max-width: 550px;
```
Comment (jamesshedden) [inline app/css/components/page-hero.scss]: @megaslug Same thing here, safe to change?

File: app/css/components/page-hero.scss [hunk 4]
```diff
@@ -12,7 +12,6 @@
   height: 100%;
   width: 100%;
   position: relative;
-  top: -38px;
```
Comment (jamesshedden) [inline app/css/components/page-hero.scss]: @megaslug Same thing here, safe to change?

File: app/css/components/page-hero.scss [hunk 5]
```diff
@@ -167,7 +166,7 @@
 }
 
 .page-hero--large {
-  height: 585px;
+  height: 790px;
```
Comment (jamesshedden) [inline app/css/components/page-hero.scss]: @megaslug Same thing here, safe to change?

File: app/pages/home/home.en.js [hunk 6]
```diff
@@ -2,206 +2,205 @@ import React from 'react';
...
+import SmallBusinessIcon from '../../icons/svg/small-business';
+import UpdateDdIcon from '../../icons/svg/update-dd';
+import ConnectTeamsIcon from '../../icons/svg/connect-teams';
+import EuropeIcon from '../../icons/svg/europe';
+import MazeFlowIcon from '../../icons/svg/maze-flow';
+import PoundRealTimeIcon from '../../icons/svg/pound-real-time';
+
+const HERO_CARDS = [
+  {
+    link: 'small_business',
```
Comment (jamesshedden) [inline app/pages/home/home.en.js]: Just noting here to make sure we add correct links eventually :)

File: app/pages/home/home.en.js [hunk 7]
```diff
@@ -2,206 +2,205 @@ import React from 'react';
...
+        <div className='u-text-center u-padding-Vxxl u-background-light-gray'>
           <div className='site-container u-padding-Txl'>
-            <div className='u-size-1of2 u-center'>
+            <div className='u-center'>
               <h2 className='u-text-heading u-color-dark-gray u-text-light u-text-l'>
-                You'll be in great company
+                Meet some of the businesses using GoCardless
               </h2>
+              <div className='u-text-center u-margin-Vl u-padding-Vs u-padding-Hxxl u-center'>
+                <img src='/images/logos/pro-logos@2x.png' />
```
Comment (jamesshedden) [inline app/pages/home/home.en.js]: @megaslug Be good to use a JPG here I think as we don't need transparency

File: app/pages/home/home.en.js [hunk 8]
```diff
@@ -2,206 +2,205 @@ import React from 'react';
...
-                Simple, transparent pricing for everyone
+                We partner with the best for effortless integrations
               </h2>
-              <p className='u-text-xs u-color-dark-gray u-margin-Vxs'>
-                Competitive rates for everyone from individuals through to large corporations
-              </p>
-              <Link to='pricing' className='btn btn--hollow u-margin-Tm'>
-                Learn about our pricing
+              <div className='u-text-center u-margin-Vm u-padding-Vs u-padding-Hxxl u-center'>
+                <img src='/images/logos/partner-logos@2x.png' />
```
Comment (jamesshedden) [inline app/pages/home/home.en.js]: @megaslug Be good to use JPG here to get file size down & as we don't need transparency here — ignore same comment that I wrote for the `pro-logos` file, which does use transparency (although could also refactor that to not need it as well separately)

File: app/css/components/page-hero.scss [hunk 9]
```diff
@@ -23,12 +22,12 @@
...
 
 .page-hero__text__desc {
-  max-width: 650px;
+  max-width: 550px;
   margin: 0 auto;
 }
 
 .page-hero--home {
-  background: $primary-color url('/images/hero/new-home-hero.jpg');
+  background: $primary-color url('/images/hero/hero-home.png');
```
Comment (megaslug) [inline app/css/components/page-hero.scss]: Yes, good point

File: app/css/components/page-hero.scss (hunk 5 above)
Comment (megaslug) [inline app/css/components/page-hero.scss]: @jamesshedden definitely not. Reverting it back and separating changes to just home.

File: app/css/components/page-hero.scss (hunk 3 above)
Comment (megaslug) [inline app/css/components/page-hero.scss]: @jamesshedden definitely not. Reverting it back and separating changes to just home.

File: app/css/components/page-hero.scss (hunk 4 above)
Comment (megaslug) [inline app/css/components/page-hero.scss]: @jamesshedden reverting it back and separating changes to just home.

File: app/pages/home/home.en.js [hunk 10]
```diff
@@ -2,206 +2,205 @@ import React from 'react';
...
-                    Ultra-low fees
+                <div className='u-padding-Txxl'>
+                  <h2 className='u-color-invert u-text-center u-text-light u-text-no-smoothing'>
+                    How can we help
+                  </h2>
+                  <div className="site-container u-margin-Vm grid">
+                    { HERO_CARDS.map((card) => {
+                      return (
+                        <div className='grid__cell u-size-1of3 u-text-center' key={ card.link }>
+                          <div className='hero-card'>
```
Comment (jamesshedden) [inline app/pages/home/home.en.js]: @megaslug Be nice to make these card elements links (rather than just the button) I reckon 🤔

**Pull request merged**